src/logger/MeetingSessionPOSTLogger.ts

```diff
@@ -102,10 +103,16 @@ export default class MeetingSessionPOSTLogger implements Destroyable {
       const batch = this.logCapture.slice(0, batchSize);
       const body = this.makeRequestBody(batch);
       try {
-        const response = await fetch(this.url, {
-          method: 'POST',
-          body,
-        });
+        const response = this.headers
```

Comment:
tip: You can write the code using one `fetch`.

```
const response = await fetch(this.url, {
  method: 'POST',
  body,
  ...(this.headers ? {
    headers: this.headers
  } : {})
});
```

Reply:
Thanks, great to know!
Updated to this format.